gym/envs/box2d/bipedal_walker.py

```diff
@@ -119,9 +119,7 @@ def EndContact(self, contact):
 class BipedalWalker(gym.Env, EzPickle):
     metadata = {"render.modes": ["human", "rgb_array"], "video.frames_per_second": FPS}
 
-    hardcore = False
-
-    def __init__(self):
+    def __init__(self, hardcore=False):
```

Comment:
Minor nitpick - could you add a type hint? 
`def __init__(self, hardcore: bool = False):`

Analogously for LunarLander. Gym doesn't have full type checks yet, but we're moving in that direction, and it's not much effort to add them here.

Reply:
Sure thing, I've added them to the corresponding environments